awswrangler/catalog/_create.py

```diff
@@ -659,7 +663,7 @@ def create_database(
     ... )
     """
     client_glue: boto3.client = _utils.client(service_name="glue", session=boto3_session)
-    args: Dict[str, str] = {"Name": name}
+    args: Dict[str, Any] = {"Name": name, **database_input_args} if database_input_args else {"Name": name}
```

Comment:
There is a line below where the database description is updated if it is different from the current value. But now we would be allowing more things to be updated so have to look at that logic again

Reply:
I guess the problematic decision is to determine what name and description to use if they are provided as arguments to the method and in the `database_input_args`

Reply:
@jaidisido `description` would still take precedence over `database_input_args['description']` and the update description condition would execute if either has a value, is there something else we need to account for?

Reply:
I was more referring to the fact that we have a logic where we update the database if the description is different. The question is whether we also want to update it if the supplied arguments in `database_input_args` are different as well...

Reply:
IMO explicit description parameter should take precedence if provided